Makefile

```diff
@@ -19,8 +22,24 @@ setup: ## Install all the build and lint dependencies
 	dep ensure
 	gometalinter --install
 
+build_all: ## Build a local snapshot binary versions of all commands
+	./scripts/build.sh ${BINARY}
+	make build_geth
+
+build_geth: ## Build a local snapshot binary version of geth. Use WITH_SVM=0 to disable building with SputnikVM (default: WITH_SVM=1)
+	$(info Building ${BINARY}/geth)
+	if [ ${WITH_SVM} == 1 ]; then ./scripts/build_sputnikvm.sh build ; else mkdir -p ./bin && go build ${LDFLAGS} -o ${BINARY}/geth ./cmd/geth ; fi
+
+install_all: ## Install all packages to $GOPATH/bin
```

Comment:
Rename to just `install`. If you see `Makefile`, you expect to be able to run `make` and then `make install` (later probably as root).